tsconfig.json

```diff
@@ -1,13 +1,23 @@
 {
   "extends": "./node_modules/gts/tsconfig-google.json",
   "compilerOptions": {
+    "lib": ["es2015", "dom"],
     "rootDir": ".",
-    "outDir": "build"
+    "outDir": "build",
+    "target": "es5",
```

Comment:
I don't think we should set `es5` here.  Lets just let that float with the default?